packages/react/src/components/TimePicker/TimePicker.test.tsx

```diff
@@ -1,58 +1,72 @@
+import '@testing-library/jest-dom';
+
 import * as React from 'react';
+import { render, cleanup, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { setWarningCallback, resetIds } from '@fluentui/utilities';
 import { TimePicker } from './TimePicker';
-import { ITimeRange, TimePickerValidationResultData } from './TimePicker.types';
-import { create } from '@fluentui/test-utilities';
-import { mount } from 'enzyme';
+import type { ITimeRange, TimePickerValidationResultData } from './TimePicker.types';
 import type { IComboBox } from '../ComboBox/ComboBox.types';
-import { KeyCodes } from '../../Utilities';
-import { render } from '@testing-library/react';
-import userEvent from '@testing-library/user-event';
 
 describe('TimePicker', () => {
+  beforeAll(() => {
+    // Prevent deprecation warnings from failing the tests
+    setWarningCallback(() => {
+      /* no-op */
+    });
+  });
+
+  afterAll(() => {
+    setWarningCallback();
+  });
+
+  afterEach(() => {
+    cleanup();
+  });
+
+  beforeEach(() => {
+    resetIds();
+  });
+
   it('renders correctly', () => {
-    const timeRange: ITimeRange = {
-      start: 0,
-      end: 5,
-    };
-    const component = create(
+    const timeRange: ITimeRange = { start: 0, end: 5 };
```

Comment:
lets undo this change as the snapshot diff is way to big to be able to tell if it's correct or not. we can keep using `renderer.create` directly